widgets/google-map/google-map.php

```diff
@@ -462,17 +464,19 @@ function get_settings_form() {
 	}
 
 	function modify_instance( $instance ) {
-		if ( empty( $instance['settings']['mobile_zoom'] ) ) {
-			// Check if a zoom is set, and if it is, set the mobile zoom to that
-			if ( empty( $instance['settings']['zoom'] ) ) {
-				$instance['settings']['mobile_zoom'] = 12;
-			} else {
-				$instance['settings']['mobile_zoom'] = $instance['settings']['zoom'];
+		if ( ! empty( $instance ) ) {
```

Comment:
This works, but it would feel "safer" to me to use `if ( ! empty( $instance['settings'] )` as it's possible for `$instance` to be a non-empty array, but not have a `settings` key which would present the same issue.